alternator/executor.cc

```diff
@@ -1358,11 +1358,27 @@ mutation put_or_delete_item::build(schema_ptr schema, api::timestamp_type ts) co
 // s_default_timeout is overwritten in alternator::controller::start_server()
 // based on the "alternator_timeout_in_ms" configuration parameter.
 db::timeout_clock::duration executor::s_default_timeout = 10s;
+db::timeout_clock::duration executor::default_getitem_timeout = 3s;
+db::timeout_clock::duration executor::default_putitem_nonlwt_timeout = 2s;
+db::timeout_clock::duration executor::default_putitem_lwt_timeout = 5s;
+db::timeout_clock::duration executor::default_query_timeout = 10s;
```

Comment:
Please explain somewhere - in a comment, in the commit message, docs/alternator or more than one of those - how you chose the different types of operations for the timeouts. Is the putitem timeouts just for putitem or also updateitem?  What applies to DeleteItem? What timeouts do BatchGetItem and BatchWriteItem use? What is the "default timeout" used for? Do operations like CreateTable, ListTable, etc., actually use any of these timeouts?